### ISSUE TITLE
Update validors and parsers to updated JSON schema

### ISSUE THREAD

**Issue by bitbeckers** [Network-Goods/hypercerts-sdk#29]
Alright! I think we finally have it!

This is what it looks like on OpenSea: https://testnets.opensea.io/assets/goerli/0xffb1fbff7a40ea441aeae8036812af116f593995/57

Updates:
- Removed the creator properties
- Promoted optional curation properties so they can be displayed on OpenSea as details
- Updated hypercert `values` and `display_values` to make the intended behavior more explicit (eg, case where `impact_scope: ["all"]`, showing that multiple scopes are conjunctive)

```
{
    "name": "Example Hypercert",
    "description": "This is where the description of the hypercert will go.",
    "external_url": "https://hypercerts.xyz",
    "image": "ipfs://bafybeifs7abhcooeelyjxmnlrcd5kuupfl5czhtyub2imzxzccrhzz3bem",
    "version": "1.0.0",
    "properties": [
        {
            "trait_type": "Example Property 1", 
            "value": "Some text here"
        },
        {
            "trait_type": "Example Property 2", 
            "value": "More text here"
        }
    ],
    "hypercert": {
        "impact_scope": {
            "name": "Impact Scope",
            "value": ["all"],
            "display_value": "All"
        },
        "work_scope": {
            "name": "Work Scope",
            "value": ["art design", "metadata standards"],
            "display_value": "Art Design & Metadata Standards"
        },
        "work_timeframe": {
            "name": "Work Timeframe",
            "value": [1663819200, 1673163072],
            "display_value": "2022-09-22 \u2192 2023-01-08"
        },
        "impact_timeframe": {
            "name": "Impact Timeframe",
            "value": [1673163072, 0],
            "display_value": "2023-01-08 \u2192 Indefinite"
        },
        "contributors": {
            "name": "Contributors",
            "value": ["0x799B774204A348E1182fE01074C51444bA70A149"],
            "display_value": "0x799...149"
        },
        "rights": {
            "name": "Rights",
            "value": ["public display", "-transfers"],
            "display_value": "Public display"
        }
    }
}
```

_Originally posted by @ccerv1 in https://github.com/Network-Goods/hypercerts/issues/98#issuecomment-1407193885_

**Comment by Jipperism** [comment]
It would also help with making sure `display_value` properties etc are formatted consistently.

**Comment by ccerv1** [comment]
> Wondering whether we want to provide a formatting utility to the end user as well, as this schema would be pretty cumbersome to format to everytime.
> It would also help with making sure `display_value` properties etc are formatted consistently.

@Jipperism agreed. I created a utility in python for rendering (see [here](https://github.com/Network-Goods/hypercerts/blob/main/utils/create_hypercert_metadata.py)) that could be adapted for this purpose

**Comment by Jipperism** [comment]
> > Wondering whether we want to provide a formatting utility to the end user as well, as this schema would be pretty cumbersome to format to everytime.
> > It would also help with making sure `display_value` properties etc are formatted consistently.
> 
> @Jipperism agreed. I created a utility in python for rendering (see [here](https://github.com/Network-Goods/hypercerts/blob/main/utils/create_hypercert_metadata.py)) that could be adapted for this purpose

Already created my own in the SDK. It might have to be updated once people settle on either "Workscope A & Workscope B" or Workscope A ^ Workscope B" for example.

**Comment by Jipperism** [comment]
https://github.com/Network-Goods/hypercerts/issues/98 linking this here

**Comment by ccerv1** [comment]
@Jipperism:
> Already created my own in the SDK. It might have to be updated once people settle on either "Workscope A & Workscope B" or Workscope A ^ Workscope B" for example.

Agreed. Happy to let @holkeb decide

**Comment by holkeb** [comment]
yes, let's use  "workscope A ∧ workscope B" 

(note that it's not ^ but ∧)
